app/controllers/proposal_controller.rb

```diff
@@ -14,6 +16,9 @@ def show
   end
 
   def new
+    @event = @program.events.new
+    @event.event_users.new(user: current_user, event_role: 'submitter') if current_user
```

Comment:
@hennevogel Am I missing something here? I can do this on master:
![screenshot from 2016-03-23 01-23-58](https://cloud.githubusercontent.com/assets/7680662/13965903/2e73cd18-f096-11e5-9888-3f071dde32a7.png)

Code [1] [2] seems to be set up this way too.

[1] https://github.com/openSUSE/osem/blob/master/app/views/proposal/new.html.haml#L25
[2] https://github.com/openSUSE/osem/blob/master/app/controllers/proposal_controller.rb#L17